### ISSUE TITLE
[Enhancement] : Enable Url link Thunderbird "Subject" column

### ISSUE THREAD

**Issue by siomosp** [fnxweb/urllink#25]
Hello, thank you for the useful extension
It is possible to enable the extension with right-click at Thunderbird  subject?
Thank you!
![urllink](https://user-images.githubusercontent.com/4071032/194815207-9b02c6f0-8504-4342-be2e-265e92690ae1.jpg)

**Comment by fnxweb** [comment]
I'll have a look.

I have a feeling it use to, back in the day, before everything got redone against WebExtensions, and when I had to port it I couldn't get that to work, but it was a while ago so it might be possible now.

**Comment by fnxweb** [comment]
No, sorry, looks like they still haven't put hooks in for extensions to be able to add context menus to the message header.  Can't find any evidence of it being looked into either :(

I'll ask on the dev. mailing list in case there's some form of hackery to do it.

**Comment by siomosp** [comment]
Thank you for considering my request :)
Maybe it can be done at messages list?
![urllink_2](https://user-images.githubusercontent.com/4071032/195333114-20c0e1c1-9029-4a1d-ba8e-b4aeb016129b.jpg)

**Comment by fnxweb** [comment]
Well, I should be able to add context menu items to their, but URL Link is driven solely off processing selected text, and you can't select text there, so not sure what the use-case would be?

**Comment by fnxweb** [comment]
I'll actually leave this open, as I had a discussion with one of the TB guys, and there's a good chance the functionality will be added as part of a header-pane rejig they're doing anyway, and I'll be able to do it (off the header selection).  He asked me to raise a TB issue so they don't forget to look at it.

Ref: https://bugzilla.mozilla.org/show_bug.cgi?id=1794755

**Comment by siomosp** [comment]
> Well, I should be able to add context menu items to their, but URL Link is driven solely off processing selected text, and you can't select text there, so not sure what the use-case would be?

Selected text can be the email subject?
(Usually in my case, the email subject is the parcel tracking number

**Comment by fnxweb** [comment]
Yes, if you select text in the subject line (in the header pane of the email view, not the list pane of all emails).  That (selection) currently isn't available to add-ons, but hopefully will be once they've done their current update of the header display code.
